src/main/java/seedu/address/model/transaction/AmountContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.model.transaction;
+
+import java.util.List;
+
+public class AmountContainsKeywordsPredicate extends WalletPredicate {
+
+    public AmountContainsKeywordsPredicate(List<String> keywords) {
+        super(keywords);
+    }
+
+    @Override
+    public boolean test(Transaction transaction) {
+        return keywords.stream()
+                .anyMatch(keyword -> (int) Double.parseDouble(keyword) ==
```

Comment:
Is it intended for the rounding to happen? Like find $/12.90 can find a transaction that is $12.50. If it's intended then it's ok!

Reply:
Oh yeaa I considered that 12 can find 12.80 or 12.30, but i forgot to consider 12.80 finding 12.50 hmm.. Sld I just make it exact?

Reply:
I made it such that when you find, only integers can be parsed!

Reply:
Up to you ba HAHA just make sure the UG is updated to explain how it works

Reply:
I think it's fine to just round it off and find values within that range (e.g. 12.30 will return all values 12.00 to 13.00), just make sure UG reflects it as Joel has mentioned!

Reply:
I made it such that only integer can be searched!